tests/api_tests/nodes/test_views.py

```diff
@@ -486,7 +526,7 @@ def test_update_public_project_logged_in(self):
         assert_equal(res.status_code, 403)
 
     def test_update_private_project_logged_out(self):
-        res = self.app.put_json(self.private_url, {
+        res = self.app.put(self.private_url, {
```

Comment:
If `put_json` is used, this happens, even though all the required fields are given:

![screen shot 2015-08-18 at 4 20 40 pm](https://cloud.githubusercontent.com/assets/9755598/9341598/de74d6d0-45c4-11e5-906e-9ade949c2eea.png)